### PR TITLE
add Dockerfile

### PR DESCRIPTION
**Pull request by lispmeister**
This Dockerfile enables the build of a Docker image for the Aperture
binary.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM golang:1.12-alpine as builder
```
Comment (guggero) [inline Dockerfile]: Should be `go1.13` as per `go.mod`.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+COPY . /go/src/github.com/lightninglabs/aperture
+
+# Force Go to use the cgo based DNS resolver. This is required to ensure DNS
+# queries required to connect to linked containers succeed.
+ENV GODEBUG netdns=cgo
+
+# Explicitly turn on the use of modules (until this becomes the default).
+ENV GO111MODULE on
+
+# Install dependencies and install/build lnd.
```
Comment (guggero) [inline Dockerfile]: nit: comment says `lnd`.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+RUN apk add --no-cache --update alpine-sdk \
+    git \
+    make \
+&&  cd /go/src/github.com/lightninglabs/aperture/cmd \
+&&  go install ./...
+
+# Start a new, final image to reduce size.
+FROM alpine as final
+
+# Expose lnd ports (server, rpc).
```
Comment (guggero) [inline Dockerfile]: nit: comment says `lnd`.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+EXPOSE 8081 11010
+
+# Copy the binaries and entrypoint from the builder image.
+COPY --from=builder /go/bin/aperture /bin/
+
+# Add bash.
+RUN apk add --no-cache \
+    bash \
+    ca-certificates
```
Comment (guggero) [inline Dockerfile]: We should probably add an `ENTRYPOINT ["aperture"]` directive to the bottom.

File: Dockerfile (hunk 1 above)
Comment (lispmeister) [reply Dockerfile]: fixed

File: Dockerfile (hunk 2 above)
Comment (lispmeister) [reply Dockerfile]: fixed

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    git \
+    make \
+&&  cd /go/src/github.com/lightninglabs/aperture/cmd \
+&&  go install ./...
+
+# Start a new, final image to reduce size.
+FROM alpine as final
+
+# Expose aperture ports
+EXPOSE 8081
```
Comment (guggero) [inline Dockerfile:24]: Port `11010` was actually correct, 8081 isn't used any more.

**Pull request merged**